src/running-the-course/translations.md

```diff
@@ -9,7 +9,7 @@ volunteers:
   [@noahdragon], [@superwhd], [@SketchK], and [@nodmp].
 - [Chinese (Traditional)][zh-TW] by [@hueich], [@victorhsieh], [@mingyc],
   [@kuanhungchen], and [@johnathan79717].
-- [Korean][ko] by [@keispace], [@jiyongp], and [@jooyunghan].
+- [Korean][ko] by [@keispace], [@jiyongp], [@jooyunghan], an [@namhyung]
```

Comment:
an -> and

Reply:
Also it needs a period at the end.

Reply:
Hi @namhyung, good catch! If you haven't seen it already, GitHub has a suggestion feature which makes it super easy to fix such things:

```suggestion
- [Korean][ko] by [@keispace], [@jiyongp], [@jooyunghan], and [@namhyung].
```